integration/examples/helm-deployment/skaffold.yaml

```diff
@@ -8,7 +8,7 @@ build:
 deploy:
   helm:
     releases:
-    - name: skaffold-helm
+    - name: skaffold-helm-{{.TEST_NS}}
```

Comment:
do we want to set this here as opposed to just setting the `namespace` field?

Reply:
actually, this will break running the examples. if I don't have this field set:

```
Starting deploy...
Error: invalid release name, must match regex ^(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])+$ and the length must not longer than 53
Helm release skaffold-helm-<no value> not installed. Installing...
No requirements found in skaffold-helm/charts.
Error: release name skaffold-helm-<no value> is invalid: a DNS-1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')
WARN[0007] error retrieving helm deployment info: Error: invalid release name, must match regex ^(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])+$ and the length must not longer than 53
```

Reply:
@nkubala Are the "examples" below `integration` really meant to be run manually by users? I think it's fine to break these, as long as the actual user facing examples in `examples` do not need the `TEST_NS` env var.

Reply:
the examples in `integration/` get synced to `examples/` on every release:

https://github.com/GoogleContainerTools/skaffold/blob/010c5179b6d4a7f77edc0482c5366743592aad58/hack/release.sh#L24-L25

so this will become a de facto example that we expect users to be able to run. I run these very frequently to test basic functionality so this will break my ability to do manual testing.

FYI the reason we have two directories and then sync them on release is so that when we merge something breaking to master, it doesn't break users trying to run their latest released version of skaffold on the `examples/` directory.

Reply:
Ooops. I always thought the examples were synced in the other direction. But that makes perfect sense. (Thanks for the explanation, I would have asked..)

Reply:
hmm, i wasn't aware we sync examples from integration to examples
I think we should skip that for this change since you can't run this integration test in parallel.

Reply:
regarding
>do we want to set this here as opposed to just setting the namespace field?

helm has this bug where release names not scoped by namespace. I think the ticket #1823 has more information.